profiles/common/modules/features/nexteuropa_feature_set/nexteuropa_feature_set.install

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * @file
+ * Implements some specific logic for platform feature sets.
+ */
+
+/**
+ * Implements hook_install().
+ *
+ * Grant access to features set and admin menu to user role.
+ */
+function nexteuropa_feature_set_install() {
+  multisite_config_service('user')->createRole('cem', 4);
```

Comment:
The cem role is needed for that feature only. Therefore it only needs to be created there. I don't think putting everything in cce_basic_config is a good pratice.
Maybe we could have a sharing session and see with JF how to better handle things